vnext/Microsoft.ReactNative/Fabric/Composition/TooltipService.cpp

```diff
@@ -256,37 +258,39 @@ void TooltipTracker::ShowTooltip(const winrt::Microsoft::ReactNative::ComponentV
     facebook::react::TextLayoutManager::GetTextLayout(
         tooltipData->attributedString, {} /*paragraphAttributes*/, layoutConstraints, tooltipData->textLayout);
 
-    DWRITE_TEXT_METRICS tm;
-    winrt::check_hresult(tooltipData->textLayout->GetMetrics(&tm));
-
-    tooltipData->width =
-        static_cast<int>(tm.width + ((tooltipHorizontalPadding + tooltipHorizontalPadding) * scaleFactor));
-    tooltipData->height = static_cast<int>(tm.height + ((tooltipTopPadding + tooltipBottomPadding) * scaleFactor));
-
-    POINT pt = {static_cast<LONG>(m_pos.X), static_cast<LONG>(m_pos.Y)};
-    ClientToScreen(parentHwnd, &pt);
-
-    RegisterTooltipWndClass();
-    HINSTANCE hInstance = GetModuleHandle(NULL);
-    m_hwndTip = CreateWindow(
-        c_tooltipWindowClassName,
-        L"Tooltip",
-        WS_POPUP,
-        pt.x - tooltipData->width / 2,
-        static_cast<int>(pt.y - tooltipData->height - (toolTipPlacementMargin * scaleFactor)),
-        tooltipData->width,
-        tooltipData->height,
-        parentHwnd,
-        NULL,
-        hInstance,
-        tooltipData.get());
-
-    DWM_WINDOW_CORNER_PREFERENCE preference = DWMWCP_ROUNDSMALL;
-    UINT borderThickness = 0;
-    DwmSetWindowAttribute(m_hwndTip, DWMWA_WINDOW_CORNER_PREFERENCE, &preference, sizeof(preference));
-
-    tooltipData.release();
-    AnimateWindow(m_hwndTip, toolTipAnimationTimeMs, AW_BLEND);
+    if (tooltipData->textLayout) {
+      DWRITE_TEXT_METRICS tm;
+      winrt::check_hresult(tooltipData->textLayout->GetMetrics(&tm));
+
+      tooltipData->width =
+          static_cast<int>((tm.width + tooltipHorizontalPadding + tooltipHorizontalPadding) * scaleFactor);
+      tooltipData->height = static_cast<int>((tm.height + tooltipTopPadding + tooltipBottomPadding) * scaleFactor);
+
+      POINT pt = {static_cast<LONG>(m_pos.X), static_cast<LONG>(m_pos.Y)};
+      ClientToScreen(parentHwnd, &pt);
+
+      RegisterTooltipWndClass();
+      HINSTANCE hInstance = GetModuleHandle(NULL);
+      m_hwndTip = CreateWindow(
+          c_tooltipWindowClassName,
+          L"Tooltip",
+          WS_POPUP,
+          pt.x - tooltipData->width / 2,
```

Comment:
`[non-blocking|commentary]` interesting, I noticed the file explorer tooltips use this pattern (centered over the mouse point, fixed distance above mouse point unless it would push the tooltip off-screen) though the Edge/webview tooltips still align the mouse point with their top-left corner.

and the minimize/maximize/close buttons in windows still use the old-school tooltips, hah.